packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -328,22 +333,16 @@ export function getDevMiniflareOptions(
 									const [moduleId] = invokePayloadData.data;
 									const moduleRE = new RegExp(MODULE_PATTERN);
 
-									// Externalize Worker modules (CompiledWasm, Text, Data)
-									if (moduleRE.test(moduleId)) {
-										const result = {
-											externalize: moduleId,
-											type: "module",
-										} satisfies vite.FetchResult;
+									const shouldExternalize =
+										// Worker modules (CompiledWasm, Text, Data)
+										moduleRE.test(moduleId) ||
+										// Node.js builtin node modules (they will be resolved to unenv aliases)
+										nodeBuiltInModules.has(moduleId);
```

Comment:
Know we've discussed it before but it still doesn't really make sense to me that we have to externalize things again here. Agree this is correct for now though.

Reply:
I 100% totally agree 👍

I'll go and investigate this 👍  (but yeah for this PR I would keep it as is since this is also effects the other modules)